filament/include/filament/Camera.h

```diff
@@ -356,17 +356,9 @@ class UTILS_PUBLIC Camera : public FilamentAPI {
      * @param center    The point in world space the camera is looking at.
      * @param up        A unit vector denoting the camera's "up" direction.
      */
-    void lookAt(const math::float3& eye,
-                const math::float3& center,
-                const math::float3& up) noexcept;
-
-    /** Sets the camera's model matrix, assuming up is along the y axis
-     *
-     * @param eye       The position of the camera in world space.
-     * @param center    The point in world space the camera is looking at.
-     */
-    void lookAt(const math::float3& eye,
-                const math::float3& center) noexcept;
+    void lookAt(math::double3 eye,
```

Comment:
Why not `const math::double3&`?

Reply:
yeah actually it's a good point, I will change it back. It looks better in more cases with references as seen in compiler explorer.